tooling/templatize/internal/ev2/mapping_test.go

```diff
@@ -0,0 +1,32 @@
+package ev2
+
+import (
+	"fmt"
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+func TestMapping(t *testing.T) {
+	testData := map[string]interface{}{
+		"key1": "value1",
+		"key2": 42,
+		"key3": true,
+		"parent": map[string]interface{}{
+			"nested": "nestedvalue",
+			"deeper": map[string]interface{}{
+				"deepest": "deepestvalue",
+			},
+		},
+	}
+	expectedFlattened := map[string]string{
+		"__KEY1__": "key1",
+		"__KEY2__": "key2",
+		"__KEY3__": "key3",
+		"__PARENT_NESTED__": "parent.nested",
+		"__PARENT_DEEPER_DEEPEST__": "parent.deeper.deepest",
+	}
+	flattened, replace := EV2Mapping(testData, []string{})
+	assert.DeepEqual(t, expectedFlattened, flattened)
```

Comment:
I know this is just a PoC - in my experience the output of `cmp.Diff()` is 1000x better than `assert.DeepEqual`

Reply:
thank you. adapted